frontend/mono_frontend/src/pages/AccountList/CardAccounts.jsx

```diff
@@ -0,0 +1,81 @@
+import { useRouteLoaderData } from 'react-router-dom'
+import { useEffect, useState } from 'react'
+import { BACKEND_URL } from '../../config/envs'
+import './CardAccounts.css'
+
+export const getCards = async function (token) {
+	const endpoint = `${BACKEND_URL}/monobank/monocards`
+
+	const response = await fetch(endpoint, {
+		method: 'GET',
+		headers: {
+			'Content-Type': 'application/json',
+			Authorization: `Bearer ${token}`
+		}
+	})
+	const data = await response.json()
+
+	if (!response.ok) {
+		return null
+	}
+	return data
+}
+
+export const CardAccounts = () => {
+	const token = useRouteLoaderData('token')
+	const [cardData, setCardData] = useState(null)
+
+	useEffect(() => {
+		const fetchData = async function () {
+			try {
+				const access = await getCards(token)
+				setCardData(access)
+			} catch (error) {
+				console.error('Error fetching card data:', error)
+			}
+		}
+
+		let ignore = fetchData()
+	}, [token])
+
+	return (
+		<>
+			{cardData && (
+				<div>
+					<table className='table'>
+						<thead className='table__head'>
+							<tr key={1} className='table__row'>
+								<th className='table__title'>№</th>
+								<th className='table__title'>AccountNumber</th>
```

Comment:
Account number, cards count and pans are not relevant in overview IMO. Lets remove them and add card type (white, black ...)

Reply:
Done